esmvaltool/diag_scripts/weighting/weighted_temperature_map.py

```diff
@@ -89,7 +89,8 @@ def mapplot(dataarray, cfg, title_pattern, filename_part, ancestors,
 def visualize_and_save_temperature(temperature: 'xr.DataArray', cfg: dict,
                                    ancestors: list):
     """Wrap mapplot: absolute temperature."""
-    title_pattern = r'Weighted {metric} temperature \n{period} ($\degree$C)'
+    title_pattern = '\n'.join(
+        ['Weighted {metric} temperature', r'{period} ($\degree$C)'])
```

Comment:
Not sure if this is now how it is supposed to be? The style checkers are happy with it but I'm not convinced...

Reply:
The second does not even make sense for such a trivial case 😅 And the first is under 80 lines, should be OK!

Reply:
If you want to split it, this is also valid python (and a cleaner way to do it):

```
    title_pattern = (r'Weighted {metric} temperature\n'
                      '{period} ($\degree$C)')
```

Reply:
Okay this is good! I need to split it otherwise I do not get the linebreak (using r'text\n')